### PR TITLE
Update INSTALLATION.md

### PR DESCRIPTION
**Pull request by UTSAVS26**
I have made some changes in `INSTALLATION.md` to make it more readable.

### CODE REVIEW

**Comment by UTSAVS26** [comment]
Hi @jrobinAV , can you please review my PR and tell what changes i need to do or it is final?

**Review by AlexandreSajus — COMMENTED** [review]
image

**Comment by UTSAVS26** [comment]
> image
> 

Thank you, i have corrected my mistake. Sorry for the trouble.

**Comment by UTSAVS26** [comment]
@FabienLelaquais @AlexandreSajus @FlorianJacta @jrobinAV please review it. Thank you

File: INSTALLATION.md [hunk 1]
```diff
@@ -1,23 +1,17 @@
-# Installation
+# Taipy Installation Guide
 
-There are different ways to install Taipy, depending on how you plan to use it.
+Taipy can be installed in various ways depending on your needs. If you aim to modify the code or contribute to its development, refer to the [source installation](#installing-for-development) section. Ensure that you have Python 3.9 or above installed on your system.
```
Comment (jrobinAV) [inline INSTALLATION.md]:
```suggestion
Taipy can be installed in various ways depending on your needs. If you aim to modify the code
or contribute to its development, refer to the 
[source installation](#installing-for-development) section. Ensure that you have Python 3.9 or
above installed on your system.
```

File: INSTALLATION.md [hunk 2]
```diff
@@ -1,23 +1,17 @@
...
 
-If your goal is to look into the code, modify and improve it, go straight
-to the [source installation](#installing-for-development) section.
+## Installing Taipy from PyPI
 
-Taipy needs your system to have Python 3.9 or above installed.
+The easiest way to install Taipy is through the [PyPI software repository](https://pypi.org/project/taipy/). You can do this by running the following command:
```
Comment (jrobinAV) [inline INSTALLATION.md]:
```suggestion
The easiest way to install Taipy is through the
[PyPI software repository](https://pypi.org/project/taipy/). You can do this by running the
following command:
```

**Review by jrobinAV — CHANGES_REQUESTED** [review]
We keep lines with less than 100 characters as much as possible for readability and maintainability.

**Comment by UTSAVS26** [comment]
> We keep lines with less than 100 characters as much as possible for readability and maintainability.

@jrobinAV made necessary changes as requested, please review. Thank you for your time.

**Review by jrobinAV — APPROVED** [review]

**Comment by UTSAVS26** [comment]
@jrobinAV Thank you for approving my PR. I am going to contribute more like this in repo.

thank you once again.

**Comment by UTSAVS26** [comment]
If possible can you add `Hacktoberfest` and `hacktoberfest-accepted` lebel if possible for you, thank you :)

**Comment by jrobinAV** [comment]
Sorry, only the PRs that refer to an `hacktoberfest` labeled issue are counting for the hacktoberfest event.

But there are [many of them](https://github.com/Avaiga/taipy/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest) not resolved yet, and we are adding some new `hacktoberfest` issues every day.

**Comment by UTSAVS26** [comment]
> Sorry, only the PRs that refer to an `hacktoberfest` labeled issue are counting for the hacktoberfest event.
> 
> But there are [many of them](https://github.com/Avaiga/taipy/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest) not resolved yet, and we are adding some new `hacktoberfest` issues every day. 

So can I create new issues for Hacktoberfest if I want to?

**Comment by jrobinAV** [comment]
1. You can create a new issue on the taipy repository. 

2. The maintainer team must review the issue and decide if it is accepted and if we want to implement it as part of the product. 

3. If the issue is accepted, the maintainer team decides whether it is open for external contribution (labeled "good first issue" or "help wanted"), whether it is a candidate for hacktoberfest event (labeled "hacktoberfest"), or if it must be implemented by an internal developer (labeled "staff only").

**Comment by UTSAVS26** [comment]
Okie @jrobinAV thanks for your clarification.

**Pull request merged**